app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/ModalFormRules.tsx

```diff
@@ -111,6 +113,11 @@ export const ModalFormRules = ({
           </Button>
           <AddOther item={item} onComplete={updateModalFromBase} />
         </div>
+        <div>
```

Comment:
Minor nit pick here. What about using a <Link> over a button here? This way an AT user would now that navigation will happen when they click/activate it.